### ISSUE TITLE
Schema: Change the Created At Date of the starter models of an instance to be the Instance Creation Date

### ISSUE THREAD

**Issue by zcolah** [zesty-io/manager-ui#1890]
When a user creates a new instance, Zesty creates some models for them by default. The key issue is that Zesty shows that these models were made 4 years ago which is basically even before the instance is made! This can freak out a user and make them think that Zesty has data integrity issues. 

Hence may we go ahead and change the created at date for these models to the date the instance was created? 

<img width="1440" alt="CleanShot 2023-02-28 at 22 12 59@2x" src="https://user-images.githubusercontent.com/22625919/222059475-60b085c5-1ced-471a-aae0-75c8116efb09.png">

**Comment by markelmad** [comment]
@agalin920 @zcolah I would agree to this request. Right now, it seems a bit of work needed to implement this. Will continue checking on how I will apply the request.
